### PR TITLE
Make changefreq editable from page

### PR DESCRIPTION
**Pull request by nelsonchuah**
Hi Dom,

Have added another feature to make sitemap's "changefreq" manageable from individual page

Thank you
Nelson

### CODE REVIEW

File: services/GoogleSitemapService.cfc [hunk 1]
```diff
@@ -176,23 +196,48 @@ component {
...
-	private function _addChildPages( required array haveAccessPages, required array childPages, string parentSearchEngineAccess, string parentAccessRestriction, string parentSitemapPriority ) {
+	private string function _getSitemapChangeFreqForPage( required string pageId ) {
+		var page = _getSiteTreeService().getPage( id=arguments.pageId, selectFields=[ "id", "parent_page", "sitemap_change_freq" ] );
+
+		if ( !page.recordCount ) {
+			return "always";
+		}
+		if ( !Len( Trim( page.sitemap_change_freq ?: "" ) ) || page.sitemap_change_freq == "inherit"   ) {
+			if ( Len( Trim( page.parent_page ) ) ) {
+				return _getSitemapPriorityForPage( page.parent_page );
```
Comment (DominicWatson) [inline services/GoogleSitemapService.cfc]: I think this should be `_getSitemapChangeFreqForPage()`?

**Review by DominicWatson — CHANGES_REQUESTED** [review]
Looking good :) I think I spotted a mistake though (see review comment). Can you ensure to test cascading, etc. in a test environment?

File: services/GoogleSitemapService.cfc (hunk 1 above)
Comment (nelsonchuah) [reply services/GoogleSitemapService.cfc]: Hi Dom,

Updated the error function call within _getSitemapChangeFreqForPage()
did a test and sitemap is generated correctly.

Thank you
Nelson

**Comment by DominicWatson** [comment]
Nice one, merging now.

**Pull request merged**